test/integration/contributionsController.test.js

```diff
@@ -0,0 +1,155 @@
+const chai = require('chai')
+const sinon = require('sinon')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const authService = require('../../services/authService')
+const githubService = require('../../services/githubService')
+const addUser = require('../utils/addUser')
+
+const app = require('../../server')
+
+chai.use(chaiHttp)
+
+// Import fixtures
+const githubPRInfo = require('../fixtures/contributions/githubPRInfo')()
+
+let jwt
+
+describe('Contributions route', function () {
+  describe('Create User', function () {
+    before(async function () {
+      const userId = await addUser()
+      jwt = authService.generateAuthToken({ userId })
+    })
+
+    afterEach(function () {
+      sinon.restore()
+    })
+
+    describe('POST /users - create new user', function () {
+      it('Should return success response after adding the user', function (done) {
+        chai
+          .request(app)
+          .post('/users')
+          .set('cookie', `rds-session=${jwt}`)
+          .send({
+            first_name: 'Prakash',
+            last_name: 'C',
+            yoe: 0,
+            img: './img.png',
+            github_id: 'prakashchoudhary07',
+            username: 'prakash'
+          })
+          .end((err, res) => {
+            if (err) { return done() }
+
+            expect(res).to.have.status(200)
+            expect(res.body).to.be.a('object')
+            expect(res.body.message).to.equal('User added successfully!')
+            expect(res.body.userId).to.be.a('string')
+
+            return done()
+          })
+      })
+    })
+  })
+  describe('POST /tasks - creates a new task', function () {
+    it('Should return success response after adding the task (isNoteWorthy = true)', function (done) {
+      chai
+        .request(app)
+        .post('/tasks')
```

Comment:
This isn't a contributions route. The tests for the tasks should be added in a separate file for tasksController.

If the motive here is to use the already added data of tasks into the contributions test, then you can simply add the data using the `tasks.updateTasks()` function ([code](https://github.com/Real-Dev-Squad/website-backend/blob/develop/models/tasks.js#L11)) in the test itself, or in before blocks and then test the response from contributions route.
This is because our actual aim here is to test the contributions routes, not the tasks route.